src/org/usfirst/frc2175/config/LoggingConfiguration.java

```diff
@@ -0,0 +1,137 @@
+package org.usfirst.frc2175.config;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.util.Properties;
+import java.util.logging.Formatter;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+import java.util.logging.SocketHandler;
+import java.util.logging.XMLFormatter;
+
+public class LoggingConfiguration extends BaseConfig {
+    /** Use ROBOT_LOCATION for deployment running. */
+    public static final String LOGGING_PROPERTIES_FILE_ROBOT_NAME =
+            "logging.properties";
+
+    /** Use ACTUAL_LOCATION for tests. */
+    public static final String LOGGING_PROPERTIES_FILE__LOCATION =
+            "src/logging.properties";
```

Comment:
@jeffjensen Is it ok to do the filenames this way, or do we want to have it based on directories instead of filenames directly? Like, have constants for `LOGGING_FILE_DIR_TEST` and `LOGGING_FILE_DIR_ROBOT`?